ui/analyse/css/_forecast.scss

```diff
@@ -32,7 +32,7 @@
     cursor: pointer;
     &:hover {
       background: $c-primary;
-      color: $c-primary-over;
+      color: $c-over;
```

Comment:
All of the $c-xxx-over colors have been collapsed to just $c-over since they were all white anyway. I discarded some semantic information here in this PR with full knowledge we may have to add some back if we ever make a new theme.